website/docs/core-concepts/vendoring/vendoring.md

```diff
@@ -0,0 +1,251 @@
+    # In 'source', Golang templates are supported  https://pkg.go.dev/text/template.
+    # If 'version' is provided, '{{.Version}}' will be replaced with the 'version' value before pulling the files from 'source'.
+    # Download the component from the AWS public ECR registry (https://docs.aws.amazon.com/AmazonECR/latest/public/public-registries.html).
+    - component: "vpc"
+      source: "oci://public.ecr.aws/cloudposse/components/terraform/stable/aws/vpc:{{.Version}}"
+      version: "latest"
+      targets:
+        - "components/terraform/infra/vpc3"
+      # Only include the files that match the 'included_paths' patterns.
+      # If 'included_paths' is not specified, all files will be matched except those that match the patterns from 'excluded_paths'.
+      # 'included_paths' support POSIX-style Globs for file names/paths (double-star `**` is supported).
+      # https://en.wikipedia.org/wiki/Glob_(programming)
+      # https://github.com/bmatcuk/doublestar#patterns
+      included_paths:
+        - "**/*.tf"
+        - "**/*.tfvars"
+        - "**/*.md"
+    - component: "vpc-flow-logs-bucket"
+      source: "github.com/cloudposse/terraform-aws-components.git//modules/vpc-flow-logs-bucket?ref={{.Version}}"
+      version: "1.323.0"
+      targets:
+        - "components/terraform/infra/vpc-flow-logs-bucket/{{.Version}}"
+      excluded_paths:
+        - "**/*.yaml"
+        - "**/*.yml"
+```
+
+<br/>
+
+- The `vendor.yaml` vendoring manifest supports Kubernetes-style YAML config to describe vendoring configuration for components, stacks,
+  and other artifacts. The file is placed into the directory from which the `atmos vendor pull` command is executed (usually the root of the repo)
+
+- The `sources` in `vendor.yaml` support all protocols (local files, Git, Mercurial, HTTP, HTTPS, Amazon S3, Google GCP), and all URL and
+  archive formats as described in [go-getter](https://github.com/hashicorp/go-getter), and also the `oci://` scheme to download artifacts from
+  [OCI registries](https://opencontainers.org).
+
+- The `targets` in the sources support absolute and relative paths (relative to the directory where the command is executed)
+
+- `included_paths` and `excluded_paths` support [POSIX-style greedy Globs](https://en.wikipedia.org/wiki/Glob_(programming)) for filenames/paths
+  (double-star/globstar `**` is supported as well)
+
+- The `component` attribute in each source is optional. It's used in the `atmos vendor pull -- component <component` command if the component is
+  passed in. In this case, Atmos will vendor only the specified component instead of vendoring all the artifacts configured in the `vendor.yaml`
+  manifest
+
+- The `source` and `targets` attributes support [Go templates](https://pkg.go.dev/text/template)
+  and [Sprig Functions](http://masterminds.github.io/sprig/). This can be used to templatise the `source` and `targets` paths with the artifact
+  versions specified in the `version` attribute
```

Comment:
```suggestion
- The `source` and `targets` attributes support [Go templates](https://pkg.go.dev/text/template)
  and [Sprig Functions](http://masterminds.github.io/sprig/). This can be used to templatize the `source` and `targets` paths with the artifact
  versions specified in the `version` attribute
 
  Here's an advanced example showcasing how templates can be used together with `targets` 
  ```yaml
      targets:
        # Vendor a component into a major-minor versioned folder like 1.2
        - "components/terraform/infra/vpc-flow-logs-bucket/{{ (first 2 (splitList \".\" .Version)) | join \".\" }}"
  ```

```